pkg/app/api/grpcapi/web_api.go

```diff
@@ -1385,5 +1385,12 @@ func (a *WebAPI) GetInsightData(ctx context.Context, req *webservice.GetInsightD
 	return &webservice.GetInsightDataResponse{
 		UpdatedAt:  updateAt,
 		DataPoints: idp,
+		Type:       model.InsightResultType_MATRIX,
+		Matrix: []*model.InsightSampleStream{
+			{
+				Labels:     nil,
```

Comment:
nit: Remove this assignment because it is not necessary.

Reply:
Thanks, fixed it!